cli/pg.go

```diff
@@ -160,11 +165,12 @@ func runPgRestore(args *docopt.Args, client controller.Client, config *runConfig
 		defer bar.Finish()
 		config.Stdin = bar.NewProxyReader(config.Stdin)
 	}
-	return pgRestore(client, config)
+
+	return pgRestore(client, config, args.String["--jobs"])
 }
 
-func pgRestore(client controller.Client, config *runConfig) error {
-	config.Args = []string{"pg_restore", "-d", config.Env["PGDATABASE"], "--clean", "--if-exists", "--no-owner", "--no-acl"}
+func pgRestore(client controller.Client, config *runConfig, jobs string) error {
```

Comment:
I did that originally but it needs to be passed as a `string` anyway so instead just opted to validate it was a valid integer by not coerce it to `int` only to need to convert it back when invoking the job.